tests/python/unittest/test_symbol.py

```diff
@@ -115,7 +115,7 @@ def test_symbol_infer_shape_var():
     shape = (2, 3)
     a = mx.symbol.Variable('a', shape=shape)
     b = mx.symbol.Variable('b')
-    c = a+b
+    c = mx.symbol.ewise_plus(a, b)
```

Comment:
this is an backward shape inference example that infers shape of b given a. Need elementwise constraint.